ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_number_of_dimensions.py

```diff
@@ -0,0 +1,75 @@
+import ivy.functional.backends.numpy as ivy_np
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+import numpy as np
+from hypothesis import given
+from hypothesis import strategies as st
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# squeeze
+@st.composite
+def _squeeze_helper(draw):
+    shape = draw(st.shared(helpers.get_shape(), key="value_shape"))
+    valid_axes = []
+    for index, axis in enumerate(shape):
+        if axis == 1:
+            valid_axes.append(index)
+    valid_axes.insert(0, None)
+
+    return draw(st.sampled_from(valid_axes))
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_dtypes,
+        shape=st.shared(helpers.get_shape(), key="value_shape"),
+    ),
+    axis=_squeeze_helper(),
+    dtype=st.sampled_from(ivy_np.valid_numeric_dtypes + (None,)),
```

Comment:
please update the tests according to the updated function args, such as `dtype` and `where` will not be required